src/webhook/types.ts

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable @typescript-eslint/ban-types */
+
+import Logger from "bunyan";
+
+type Hooks = {
+	[key: string]: Function[];
+};
+
+export interface State {
+	hooks: Hooks;
+}
+
+export class WebhookContext {
```

Comment:
Not much point using a class for a simple object that holds data.  Could just use an interface for this.

Reply:
We're going to replace Probot context's class with WebhookContext. We've used various context's class method such as repo in our exisiting code, so WebhookContext is going to have similar methods.